react/components/OrderInfo/CustumerInfo.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const CustumerInfo = ({ profile }) => {
+  return (
+    <div className="flex flex-column">
```

Comment:
Nitpick:
```
const CustumerInfo = ({ profile }) => (
     <div className="flex flex-column">
```

Reply:
Updated all functional components (pretty much all components hahahah) to follow this pattern. It does look cleaner :)